text/0085-api-keys.md

```diff
@@ -0,0 +1,550 @@
+
+##### 1.5.3.1 Default Search API Key
+
+The `Default Search API key` gives access to the same rights as the old `public` key.
+
+Here is how the `Default Search API Key` is represented after its generation.
+
+```json
+{
+    "description": "Default Search API Key (Use it to search from the frontend)",
+    "key": "0a6e572506c52ab0bd6195921575d23092b7f0c284ab4ac86d12346c33057f99", //example
+    "actions": [
+        "search"
+    ],
+    "indexes": [
+        "*"
+    ],
+    "expiresAt": null,
+    "createdAt": "2021-08-11T10:00:00Z", //example
+    "updatedAt": "2021-08-11T10:00:00Z"
+}
+```
+
+##### 1.5.3.2 Default Admin API Key
+
+The `Default Admin API key` gives access to the same rights as the old `private` key.
+
+Here is how the `Default Admin API Key` is represented after its generation.
+
+```json
+{
+    "description": "Default Admin API Key (Use it for all other operations. Caution! Do not use it on a public frontend)",
+    "key": "380689dd379232519a54d15935750cc7625620a2ea2fc06907cb40ba5b421b6f", //example
+    "actions": [
+        "*"
+    ],
+    "indexes": [
+        "*"
+    ],
+    "expiresAt": null,
+    "createdAt": "2021-08-11T10:00:00Z", //example
+    "updatedAt": "2021-08-11T10:00:00Z"
+}
+```
+
+#### 1.5.3 Managing `API Key`
+
+![](https://i.imgur.com/mAUFnNb.png)
+
+`Anna` manages the MeiliSearch instance; she uses the master key she defined at startup to generate an `API Key` resource for `Mark` to use in his client code to communicate with the MeiliSearch instance.
+
+`Anna` can define access rights to certain indexes to define an expiration date and also authorized `actions` for an `API Key` (See API Key Actions List Definition Part).
+
+Only the master key allows managing the API keys.
+
+#### 1.5.4 `API Key` object representation
+
+| field       | type    | description                                      |
+|-------------|---------|--------------------------------------------------|
+| description | string  | A description for the key. `null` if empty. |
+| key         | string  | The generated key. **Generated by MeiliSearch**. |
+| actions     | array   | A list of actions permitted for the key. `["*"]` for all actions. See Actions list definition part. |
+| indexes     | array   | A list of indexes permitted for the key. `["*"]` for all indexes. |
+| expiresAt   | string  | Represent the expiration date and time as `ISO-8601` format. `null` equals to no expiration time. |
+| createdAt   | string  | Represent the date and time as `ISO-8601` format when the API key has been created. **Generated by MeiliSearch** |
+| updatedAt   | string  | Represent the date and time as `ISO-8601` format when the API key has been updated. **Default**: Value of `createdAt`. **Generated by MeiliSearch** |
+
+#### 1.5.5 `POST`/ `PATCH` - `/keys` - API Key object payload definition
+
+| field       | type    | required |description                     |
```

Comment:
We could add a new column here to list "Default" what do you think? 🧐

Reply:
Absolutely, and it can be done every time we need to describe some API resources object or payload.